test/async-pingpong/test.c

```diff
@@ -149,7 +164,10 @@ child_main(void) {
   mtev_dso_post_init();
 
   mtev_conf_write_log();
+#pragma GCC diagnostic push
```

Comment:
please provide warning

Reply:
This will be the same fix we settled on for the pool-shift-async test.